stl/CMakeLists.txt

```diff
@@ -262,6 +262,7 @@ set(IMPLIB_SOURCES
     ${CMAKE_CURRENT_LIST_DIR}/src/syserror_import_lib.cpp
     ${CMAKE_CURRENT_LIST_DIR}/src/vector_algorithms.cpp
     ${CMAKE_CURRENT_LIST_DIR}/src/xonce2.cpp
+    ${CMAKE_CURRENT_LIST_DIR}/src/alias.asm

```

Comment:
Any reason not to keep these lexicographically ordered?

Reply:
not really, except that it's a different language, so this makes it easier to pull out asm files (in msbuild they will need to go in a seperate ItemGroup I think